__tests__/app.test.js

```diff
@@ -0,0 +1,23 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+
+seed(testData);
+
+describe.only("CORE: GET /api/topics", () => {
+  it("responds with an array of topic objects - with properties slug and description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toHaveLength(3);
+        body.forEach((topic) => {
+          expect(topic).toMatchObject({
+            slug: expect.any(String),
+            description: expect.any(String),
+          });
```

Comment:
Great combination of toMatchObject and expect.any() here to check each item in your array